ansible/roles/common/templates/rails_start.sh.j2

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "Starting the Dryad rails server..."
+cd ~/dryad
+rails server --binding=0.0.0.0 
```

Comment:
Can you add `"$@"` to the end of the `rails server` call, to propagate any extra arguments one might pass? For example, `-e local` starts the local environment instead of development.

Reply:
I verified that this will work on my test machine.